src/app/common/components/flows-table/flows-table.component.ts

```diff
@@ -196,6 +208,57 @@ export class FlowsTableComponent implements OnInit, OnChanges {
         return this.tableOptions.displayColumns.includes("dataset");
     }
 
+    public onForceUpdate(node: FlowSummaryDataFragment): void {
+        if (
+            node.description.__typename === "FlowDescriptionDatasetPollingIngest" &&
+            node.configSnapshot?.__typename === "FlowConfigurationIngest"
+        ) {
+            this.setScheduleOptions(node);
+            this.datasetFlowsService
+                .datasetTriggerFlow({
+                    datasetId: node.description.datasetId,
+                    datasetFlowType: DatasetFlowType.Ingest,
+                    flowRunConfiguration: {
+                        ingest: this.scheduleOptions,
+                    },
+                })
+                .pipe(takeUntilDestroyed(this.destroyRef))
+                .subscribe((result: boolean) => {
```

Comment:
Error handling?

Reply:
Error handling exists in the service method

![image](https://github.com/user-attachments/assets/300d121d-eb3a-4afb-a04e-d70b94d3e9ee)